.ci/kitchen-windows2016-py2

```diff
@@ -12,7 +12,16 @@ def python_version = 'py2'
 def test_transport = 'ZeroMQ'
 def salt_target_branch = 'neon'
 def golden_images_branch = 'neon'
-def nox_passthrough_opts = '--log-cli-level=warning --ignore=tests/utils'
+
+// While we don't move the Branch job to Pytest
+def nox_passthrough_opts
+def tests_runner = 'runtests'
+
+if (tests_runner == 'runtests') {
+    nox_passthrough_opts = '--unit'
```

Comment:
Does this mean we will only be running unit tests on windows?

Reply:
No, the nox kitchen verifier will add the `--from-files` white list too